apps/meteor/client/views/room/contextualBar/Threads/components/ThreadChat.tsx

```diff
@@ -110,7 +110,13 @@ const ThreadChat = ({ mainMessage }: ThreadChatProps) => {
 				>
 					<Field>
 						<Field.Row marginBlock={8}>
-							<CheckBox id={sendToChannelID} checked={sendToChannel} onChange={() => setSendToChannel((checked) => !checked)} />
+							<CheckBox
+								id={sendToChannelID}
+								checked={sendToChannel}
+								onChange={() => setSendToChannel((checked) => !checked)}
+								role='checkbox'
```

Comment:
I added `role='checkbox'` to grab this DOM node on playwright tests, we could have done this in other ways but I went with this as this would also improve accessibility. 

I have now removed the role checkbox, Please take a look.